search/searcher/search_phrase.go

```diff
@@ -196,7 +258,7 @@ func findPhrasePaths(prevPos uint64, ap search.ArrayPositions, phraseTerms []str
 	cdr := phraseTerms[1:]
 
 	// empty term is treated as match (continue)
-	if car == "" {
+	if len(car) == 0 || len(car) == 1 && car[0] == "" {
```

Comment:
yep, i agree – i'm typically in favor of more parentheses.  i think i added this in at the last minute and didn't really look at it very carefully.